sample_records/record.yaml

```diff
@@ -43,31 +38,24 @@ time_coverage_end: "1985-04-25T12:00:00Z"
 time_coverage_duration: "P1D"
 time_coverage_resolution: "P1D"
 standard_name_vocabulary: "standard_name_vocabulary"
-license: "license in french"
-license_eng: "license in english"
-
 contributor_name: "contributor_name"
 contributor_role: "contributor_role"
 publisher_name: "publisher_name"
 publisher_url: "publisher_url"
 publisher_email: "publisher_email"
 date_modified: "2019-10-07T21:27:22Z"
 date_issued: "2010-01-24"
-geospatial_lat_units: "degrees_north"
```

Comment:
These haven't been used by the template so I removed them, can always be added back if we want them